ui/app/templates/csi/volumes/volume.hbs

```diff
@@ -1,128 +1,193 @@
+{{#each this.breadcrumbs as |crumb|}}
+  <Breadcrumb @crumb={{crumb}} />
+{{/each}}
 {{page-title "CSI Volume " this.model.name}}
 <section class="section with-headspace">
-  <h1 class="title" data-test-title>{{this.model.name}}</h1>
-
+  <h1 class="title" data-test-title>
+    {{this.model.name}}
+  </h1>
   <div class="boxed-section is-small">
     <div class="boxed-section-body inline-definitions">
-      <span class="label">Volume Details</span>
+      <span class="label">
+        Volume Details
+      </span>
       <span class="pair" data-test-volume-health>
-        <span class="term">Health</span>
+        <span class="term">
+          Health
+        </span>
         {{if this.model.schedulable "Schedulable" "Unschedulable"}}
       </span>
       <span class="pair" data-test-volume-provider>
-        <span class="term">Provider</span>
+        <span class="term">
+          Provider
+        </span>
         {{this.model.provider}}
       </span>
       <span class="pair" data-test-volume-external-id>
-        <span class="term">External ID</span>
+        <span class="term">
+          External ID
+        </span>
         {{this.model.externalId}}
       </span>
       {{#if this.system.shouldShowNamespaces}}
         <span class="pair" data-test-volume-namespace>
-          <span class="term">Namespace</span>
+          <span class="term">
+            Namespace
+          </span>
           {{this.model.namespace.name}}
         </span>
       {{/if}}
     </div>
   </div>
-
   <div class="boxed-section">
     <div class="boxed-section-head">
       Write Allocations
     </div>
     <div class="boxed-section-body {{if this.model.writeAllocations.length "is-full-bleed"}}">
       {{#if this.model.writeAllocations.length}}
-        <ListTable
-          @source={{this.sortedWriteAllocations}}
-          @class="with-foot" as |t|>
+        <ListTable @source={{this.sortedWriteAllocations}} @class="with-foot" as |t|>
           <t.head>
             <th class="is-narrow"></th>
-            <th>ID</th>
-            <th>Created</th>
-            <th>Modified</th>
-            <th>Status</th>
-            <th>Client</th>
-            <th>Job</th>
-            <th>Version</th>
-            <th>CPU</th>
-            <th>Memory</th>
+            <th>
+              ID
+            </th>
+            <th>
+              Created
+            </th>
+            <th>
+              Modified
+            </th>
+            <th>
+              Status
+            </th>
+            <th>
+              Client
+            </th>
+            <th>
+              Job
+            </th>
+            <th>
+              Version
+            </th>
+            <th>
+              CPU
+            </th>
+            <th>
+              Memory
+            </th>
```

Comment:
I find that this (and other table header changes) make the file harder to read. Is this something that can be disabled in your code formater?

Reply:
I have major issues with my eslint and prettier config. I'm planning to tackle this in one swoop later. But, I can spend the day tomorrow and figure this out.

Reply:
Cool, let me know if you need help. It may be easier to do it with an interactive rebase with git than manually revert them.